### ISSUE TITLE
Totems - why?

### ISSUE THREAD

**Issue by chrisbecke** [Azanor/thaumcraft-suggestions#31]
Perhaps I have an unusual play style, but ... I can't see the point of ever creating any of the aura dispersal or attraction totems.

Given:
The use of totems, and 'other' thaumaturgy will contribute to flux levels over time,
High levels of flux will cause recurring taint issues and corrupt all vis shards into vitium shards,
The only way to combat flux is the use of pure nodes,
the pure nodes created are small, and get used up:
The only sane response is to plant many silverwoods to get some pure nodes.
As a consequence, the aura in the protected area is getting a constant supply of new aura, and has no vitium buildup.
As a consequence, there is no need to either attract useful vis, or repel vitium.

**Comment by ThePB** [comment]
There is use in attracting vitium and repelling vis, though. Not all the areas in the overall vicinity of your protected area are going to have an high level of aura constantly unless you somehow force vis to travel and the easiest way is to have totems do the work for you (at least until you can plant lots of silverwood everywhere and not everyone is lucky with silverwood drops)

And, unless i remember wrongly, Totems do not create flux but convert moved vis into flux, which is a quite substantial difference.

**Comment by chrisbecke** [comment]
If vitium was needed, it seems far easier to make, than attract. If it is being attracted for the purpose of clearing, it seems foolish to generate more as part of the process.
I cannot imagine why one would want to repel normal vis.
There is also no thaumcraft mechanic that requires consistently high levels of aura  - save for crystal growth - but there you simply grow the crystals in the chunks that contain the relevant nodes. All other crafting can rely on the flow of vis from high concentration to low.
As to silverwood drops - this can be volatile, but the process is positive - a tree on average drops slighly more than 1 sapling so if you have enough saplings to start with to cover the volatility you can plant an infinite number of saplings if you have the patience to cut them all down.
On the last part - does this mean that there is no negative effect to either attracting or repelling vitium?

**Comment by ThePB** [comment]
> On the last part - does this mean that there is no negative effect to either attracting or repelling vitium?

Exactly. At least while i was testing a long time ago in a galaxy... ok no let's not go there :d
But you know, i have no "proof" except dioptra and thaumometer levels, so not really 100% sure there. No code confirmation. But lots of tainted shards used and no relevant increase of vitium level (nor decrease, if you are wondering)

And regarding silverwood... the point of chance is that it's not guaranteed. I know, in the long run it's going to end up being not a problem, but starting off it's still an issue that CAN manifest itself and totems DO partially cover this with their reliability. I had problems more than once to even gathering a SINGLE silverwood sapling. It just happens (well, less so recently, and still having a full formation of nodes to get the aura completely going is a task that takes time, expecially in single player)

Why do you want to have "high" aura? because i want to be able to fly with the harness, regen my shielding, repair my equipment EVEN when i'm outside of my zone of confort while not having to worry about draining the zone. Spreading aura from a place where it's plentiful  to cover an area where it's not can be useful (ie le random node in the middle of nowhere near the stronghold that i have to reach to go into the end).
Also increases natural crystal growth, and that can help in preventing dangeroulsy low level of vis of specific types - while also forming a barrier (via possible crystal conversion) to random taint events.

They are not "the solution to every known problem", but they have their uses.

lore: I suppose that that was another possible function of those obsidian totems of old.

(btw, still feeling that something could be done to make them even more interesting... something along a "void transferral pillar" that allows 2 totems to link and work very long distances, one attracting vis in one place and the other dispersing in another... or possibly even working trans-dimensionally....)

**Comment by MalkContent** [comment]
I do like the totems, but sometimes I feel like they are an ineffective way to move vis around.
I could go for getting the TC4 laser beams back. Feed the laser with totems of attraction, beam the laser to a totem of dispersal, et voil.
Upsides: increased control, wee lasers, pretty!
Downsides: Azanor and everyone experiencing problems with them going "fml, lasers are back".
